longevity_tombstone_gc_test.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+import time
+from functools import partial
+
+from longevity_twcs_test import TWCSLongevityTest
+from sdcm.utils.common import ParallelObject
+from sdcm.utils.sstable_utils import SstableUtils
+
+
+class TombstoneGcLongevityTest(TWCSLongevityTest):
+    keyspace = 'scylla_bench'
+    table = 'test'
+    ks_cf = f'{keyspace}.{table}'
+    propagation_delay = 60 * 4  # setting a value shorter than the default, of 4 minutes
+    repair_date = None
+    db_node = None
+
+    def _run_repair_and_major_compaction(self, wait_propagation_delay: bool = False):
+        self.log.info('Run a flush for user-table on nodes')
```

Comment:
it is running on the specific keyspace `keyspace`...  you can change it to
```suggestion
        self.log.info(f'Run a flush for {self.keyspace=} on nodes')
```

Reply:
fixed